test/rfc6070-vectors.config

```diff
@@ -0,0 +1,28 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

Reply:
I'm not sure. Do static analysis fixes ever apply to test data?